### PR TITLE
split into main and mcshub

### PR DESCRIPTION
**Pull request by elevran**
Signed-off-by: Etai Lev Ran <etail@il.ibm.com>

### CODE REVIEW

File: cmd/hub/mcscontroller/main.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ limitations under the License.
 // The controller is responsible for coordinating a common view of exports
 // and imports on a shared (a.k.a. hub) cluster, and replaces the, per-workload-
 // cluster mapping, currently provided by Submariner's lighthouse component.
-package mcscontroller
+package main
```
Comment (davidohana) [inline cmd/hub/mcscontroller/main.go:24]: shall the parent folder be called  `mcshub` as well as in the pkg folder?

Comment (davidohana) [reply cmd/hub/mcscontroller/main.go:24]: or maybe put the main under `pkg/mcshub` like the agent's structure, for coherency?

Comment (elevran) [reply cmd/hub/mcscontroller/main.go:24]: can't have main and the controller test files in the same directory
(unable to import package main which is required for executable)
Renamed from `cmd/hub/mcscontroller` to `cmd/mcshub`

Comment (davidohana) [reply cmd/hub/mcscontroller/main.go:24]: in the agent they put the controller under controller subdir ..

**Review by davidohana — APPROVED** [review]

**Pull request merged**